oneview_redfish_toolkit/app.py

```diff
@@ -100,6 +108,35 @@
     app.register_blueprint(network_port)
     app.register_blueprint(session)
 
+    @app.before_request
+    def check_authentication():
```

Comment:
Missing documentation for `check_authentication()`.

Reply:
fixed